gulp/tasks/pldoc_scripts.js

```diff
@@ -17,3 +17,14 @@ gulp.task('pldoc_scripts', ['pldoc_scripts-lint'], function() {
         .pipe(gulp.dest(config.pldoc_dest))
         .pipe(browserSync.reload({stream:true}))
 });
+
+gulp.task('pldoc_scripts', ['pldoc_scripts-lint'], function() {
+    return gulp.src([
+        // setup sequence for files that are loaded later with RequireJS
+        './pattern-library/js/*'
```

Comment:
Yeah, I would try to make this into a configuration variable. I think ``pldoc_src_files`` makes sense if you want to sync it up with your pathing change or if you want to try to convert/simplify and just make ``pldoc_src`` this value.